app/hooks/scenarios/index.ts

```diff
@@ -1140,3 +1140,41 @@ export function useDeletePUScenaro() {
     },
   });
 }
+
+export function useDownloadSolutionsSummary({
+  requestConfig = {
+    method: 'GET',
+  },
+}) {
+  const { data: session } = useSession();
+
+  const downloadScenarioSolutionsSummary = ({ id }: { id: Project['id'] }) => {
+    return DOWNLOADS.request({
+      url: `/projects/${id}/output-summary`,
+      responseType: 'arraybuffer',
+      headers: {
+        Authorization: `Bearer ${session.accessToken}`,
+        'Content-Type': 'application/zip',
+      },
+      ...requestConfig,
+    });
+  };
+
+  return useMutation(downloadScenarioSolutionsSummary, {
+    onSuccess: (data, variables) => {
+      const { data: blob } = data;
+      const { id } = variables;
+
+      const url = window.URL.createObjectURL(new Blob([blob]));
+      const link = document.createElement('a');
+      link.href = url;
+      link.setAttribute('download', `solutions-summary-${id}.zip`);
+      document.body.appendChild(link);
+      link.click();
+      link.remove();
```

Comment:
I think I've seen this exact code in several places... At this rate, I think we can extract it to a utility and just reference it.